lib/services/nodes-api-service.js

```diff
@@ -280,6 +280,11 @@ function nodeApiServiceFactory(
                     }
                 };
                 return workflowApiService.createAndRunGraph(configuration);
+            } else if (node.type === Constants.NodeTypes.Rack) {
+                return eventsProtocol.publishNodeEvent(node, 'added')
```

Comment:
I can do that, but the "added" event will only represent/trigger for nodes added through posts to the api.